spec/inputsOutputs/inputsOutputs.spec.js

```diff
@@ -163,7 +168,6 @@ describe('inputsOutputs page', function() {
       expect(components.layout.isElementDisplayed(element(by.css('.popover')))).toBe(true);//popover should show up
 
       components.popovers.popover.clickYes();//push 'Yes' btn
-      expect(components.layout.isElementPresent(element(by.css('.popover')))).toBe(false);//popover should be hide
```

Comment:
we should test clicking no, replace "isElementDisplayed" with protractor isPresent method.